content/guides/overview/why-cypress.md

```diff
@@ -147,6 +148,13 @@ the developer tools you know and love.
 
 <DocsVideo src="/img/snippets/debugging.mp4"></DocsVideo>
 
+### <Icon name="chart-bar"></Icon> Analyzing test results
+
+The Cypress Dashboard collects details of your test runs and gives you detailed
+logs and analytics that help you surface issues and combat flake.
+
+<DocsVideo src="/img/snippets/debugging.mp4"></DocsVideo>
```

Comment:
This needs to be replaced with the new dashboard snippet once it's been done.

Reply:
Nothing

Reply:
Cheers m8